frontend/src/main/scala/bloop/engine/Feedback.scala

```diff
@@ -93,6 +93,13 @@ object Feedback {
   def unknownHostName(host: String): String =
     s"Host name '$host' could not be either parsed or resolved"
 
+  def skippedUnsupportedScalaMetals(scalaVersion: String): String =
+    s"Skipped configuration of SemanticDB in unsupported $scalaVersion projects"
+  def configuredMetalsProjects(projects: Traversable[Project]): String =
+    s"Configured SemanticDB in projects ${Project.pprint(projects)}"
+  def failedMetalsConfiguration(version: String, cause: String): String =
+    s"Stopped configuration of SemanticDB in Scala $version projects: $cause"
```

Comment:
I usually add user-facing messages in `Feedback` objects so that I can easily know where to change a log message if I don't like it and so that it removes the distraction of reading what the output to the users is from the call-sites.